webpack.config.js

```diff
@@ -1,22 +1,22 @@
 const path = require('path');
 
 module.exports = {
-  entry: ['./browser/src/react-redux/index.jsx'],
+  entry: ['./browser/src/react-redux/index.react.js'],
   mode: 'development',
   module: {
     rules: [{
-      test: /jsx?$/,
+      test: /js?$/,
       exclude: /node_modules/,
       use: {
         loader: 'babel-loader',
       },
     }],
   },
   resolve: {
-    extensions: ['.js', '.jsx'],
+    extensions: ['.js'],
   },
   output: {
-    path: path.join(__dirname, '/browser/public/js'),
+    path: path.resolve(__dirname, '/browser/public/js'),
```

Comment:
Just curious, why did you change this to resolve? I'm not against it, just curious. I actually don't really know the difference between the 2.

Reply:
Update, I actually experienced a permissions error with this because we need to exclude the `/` before browser. Double check that removing that forward slash still works on Windows.

Reply:
FYI, it will bump it to the next line if you put the backticks on separate lines from the code.

```
path: path.resolve(__dirname, 'browser/public/js'),
```

Reply:
oops sorry. I rushed this. My understanding of this is both methods concatenate the directories with the parentheses. However, `path.join` creates a relative path and `path.resolve` creates an absolute path. Because `__dirname` is at the beginning for both, you end up with the same result. I think join makes more sense though.  

Should I make webpack.config changes in the `87-scss-modules-webpack-config` branch? There is plenty of other changes to webpack.config there.